test/functional/cmdLineTests/sigxfszHandlingTest/sigxfszHandlingTest.xml

```diff
@@ -84,7 +84,7 @@
  </test>
 
  <test id="-XX:+HandleSIGXFSZ -Xrs">
-	<command>$EXE$ -Xshareclasses:none $CP$ $HANDLE$ -Xrs j9vm.test.sigxfsz.SignalXfszTest test.txt</command>
+	<command>$RUN_SCRIPT$ $TESTDIR$$PATHSEP$runSIGXFSZTest$SCRIPT_SUFFIX$ "$EXE$ -Xshareclasses:none $CP$ $HANDLE$ -Xrs j9vm.test.sigxfsz.SignalXfszTest test.txt"</command>
```

Comment:
Some of the tests are platform limited like the one below - `platforms="linux.*">`.  I'm assuming these tests only run on unix platforms so a `!#/bin/sh` script is valid.

Reply:
Yes, these tests only run on Unix platforms. So,`!#/bin/sh` is valid. Checked a AIX and a zOS machine to confirm `!#/bin/sh`. Existing test code for Unix platforms also utilizes `!#/bin/sh`. Example: 
https://github.com/eclipse/openj9/blob/451b01ecbe5276c0b566faae9adda9338c5d950e/test/functional/cmdLineTests/URLClassLoaderTests/Sanity/buildVerify.sh#L1